ee/codegen/src/generators/nodes/search-node.ts

```diff
@@ -460,45 +477,15 @@ export class SearchNodeMetadataFilters extends AstNode {
         }),
         python.methodArgument({
           name: "lhs_variable",
-          value: python.instantiateClass({
-            classReference: python.reference({
-              name: "StringVellumValueRequest",
-              modulePath: [...VELLUM_CLIENT_MODULE_PATH, "types"],
-            }),
-            arguments_: [
-              python.methodArgument({
-                name: "type",
-                value: python.TypeInstantiation.str("STRING"),
-              }),
-              python.methodArgument({
-                name: "value",
-                value: python.TypeInstantiation.str(lhsId),
-              }),
-            ],
-          }),
+          value: lhs,
```

Comment:
Here's where we codegen the descriptor instead of the id of the node input that represents the descriptor.